contributor_docs/es/steward_guidelines.md

```diff
@@ -0,0 +1,384 @@
+4. Si el error se origina en el código que el usuario proporcionó en el informe de error y no en el comportamiento de p5.js:
+   - Determine si la documentación de p5.js, la implementación de código o el sistema de errores amigable pueden mejorarse para evitar que se cometa el mismo error.
+   - Redirija amablemente cualquier pregunta adicional al foro [forum](https://discourse.processing.org/)  o al Discord [Discord](https://discord.com/invite/SHQ8dH25r9) y cierre el issue si no se van a realizar más cambios en p5.js.
+   
+
+
+### Solicitud de función (Feature request)
+
+Los issues/problemas de solicitudes de función (Feature request issues) deberían utilizar la plantilla de problema(issue) "Nueva Solicitud de Función" ("New Feature Request") . El siguiente flujo de trabajo es típico para abordar las solicitudes de función:"
+
+1. Como parte del compromiso de p5.js de aumentar el acceso, una solicitud de función(feature request) debe justificar cómo aumenta el acceso de p5.js a comunidades que históricamente han sido marginadas en el campo. Más detalles están disponibles aquí[here](access.md).
+   - If a feature request does not have the "Increasing Access" field sufficiently filled out, you can ask the issue author how the feature increases access.
+   - Si una solicitud de función(feature request) no tiene suficientemente completado el campo "Aumento de Acceso"("Increasing Access"), puedes preguntar al autor del problema(issue autor) cómo la función(feature) aumenta el acceso.
+   - La declaración de acceso de una función puede ser proporcionada por un miembro diferente de la comunidad, incluidos los revisores de problemas(issue reviewers).
+1. La nueva solicitud de función puede ser evaluada para su inclusión en base a los siguientes criterios:
+   - ¿La función(feature) encaja en el alcance del proyecto y los principios de diseño [design principles](design_principles.md) de p5.js?
+     - Por ejemplo, una solicitud para agregar una nueva forma primitiva de dibujo puede ser considerada, pero una solicitud para adoptar un protocolo de Internet de las cosas basado en el navegador (browser-based IOT protocol) probablemente estará fuera de alcance.
+     - En general, el alcance de p5.js debería ser relativamente estrecho para evitar un exceso de características poco utilizadas.
+     - Si una función no encaja en el alcance de p5.js, sugiere al autor del problema(issue) que implemente la función(feature) como una biblioteca complementaria(addon library).
+     - Si no está claro si encaja o no, puede ser una buena idea sugerir hacer una biblioteca complementaria(addon library)como una prueba de concepto(proof-of-concept).Esto ayuda a dar a los usuarios una forma de usar la función(feature), proporciona un ejemplo mucho más concreto de su uso e importancia, y no necesariamente necesita ser una solución tan completa como una función completamente integrada. Puede integrarse en el núcleo(the core) de p5.js más adelante si corresponde. 
```

Comment:
Hay muchas instancias donde es inconsistente el uso de espacios entre parentesis y puntos, por ejemplo:
"..rary)como una prueba de concepto(proof-of-concept).Es.."
Por favor corregir

Reply:
esto no se ha corregido